core/src/main/java/org/kiwix/kiwixmobile/core/main/OnSwipeTouchListener.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.main
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.view.GestureDetector
+import android.view.GestureDetector.SimpleOnGestureListener
+import android.view.MotionEvent
+import android.view.View
+import android.view.View.OnTouchListener
+import kotlin.math.abs
+
+open class OnSwipeTouchListener @SuppressLint("SyntheticAccessor")
+constructor(ctx: Context?) : OnTouchListener {
```

Comment:
And this change should not be present on this PR, to pair my comment with your other PR

Reply:
When you said to change OnswipeTouchListener so I change The code . When I change the code of ToolbarScrollingKiwixWebView then when I committed both the files get included.

Reply:
You should do separate tickets on separate branches so they can be reviewed and altered separately

Reply:
What I do now Sir.

Reply:
That depends very much on your git history. It is probably easiest if start over and work on branches instead of develop, to enforce good behaviour if nothing else

Reply:
Okay sir